src/main/frontend/handler/editor.cljs

```diff
@@ -984,11 +984,16 @@
 (defn- compose-copied-blocks-contents-&-block-tree
   [repo block-ids]
   (let [blocks (db-utils/pull-many repo '[*] (mapv (fn [id] [:block/uuid id]) block-ids))
-        unordered? (:block/unordered (first blocks))
-        format (:block/format (first blocks))
-        level-blocks-map (blocks-with-level blocks)
+        blocks* (flatten
+                 (mapv (fn [b] (if (:collapsed (:block/properties b))
+                                 (vec (tree/sort-blocks (db/get-block-children repo (:block/uuid b)) b))
+                                 [b])) blocks))
+        block-ids* (mapv #(:block/uuid %) blocks*)
```

Comment:
`:block/uuid` can be used as a function directily

Reply:
fixed